packages/dx-scheduler-core/src/types/horizontal-rect.types.ts

```diff
@@ -0,0 +1,40 @@
+import { PureComputed } from '@devexpress/dx-core';
+import { Multiline, CellElement, AppointmentDate } from './scheduler-core.types';
+import { ViewCellData, TakePrevious } from './all-day-panel.types';
+import { EndDate } from './appointment-form.types';
+
+export interface HorizontalPayload {
+  multiline: Multiline;
```

Comment:
Can we use 'boolean' instead of 'Multiline'?

Common question: do types like `export type Multiline = boolean;` make any sense?

Reply:
At the same time, we use 'today: boolean;' in the 'MonthCellData' interface.